examples/vue-notification-center-example/.env.example

```diff
@@ -0,0 +1,2 @@
+VITE_NOVU_APP_IDENTIFIER=YOUR_APP_IDENTIFIER
+VITE_NOVU_SUBSCRIBER_ID=YOUR_SUBSCRIBER_ID
```

Comment:
in order to run the app you should copy this `.env.example` file to `.env` and fill in the correct data

Reply:
Can we make a helper script about it like we have in the root of Novu?

Reply:
I'll update that script to populate `.env` for example apps